templates/pages/admin/form/access_control/allow_list.html.twig

```diff
@@ -56,18 +56,20 @@
 
 <script>
     // Each time the dropdown is modified, populate the helper link
-    $("select[name='_allow_list_dropdown[]']").on('change', async function() {
-        const response = await $.post(
-            CFG_GLPI.root_doc + '/ajax/form/allow_list_dropdown_count_users.php',
+    const $select = $("select[name='_access_control[{{ access_control.getId() }}][_allow_list_dropdown][]']")
```

Comment:
Selector was broken after some changes done by another PR.
Added an e2e test to make sure it doesn't break again.